tests/test_c_generator.py

```diff
@@ -1,11 +1,15 @@
+import os
+import platform
 import sys
 import textwrap
 import unittest
 
 # Run from the root dir
 sys.path.insert(0, '.')
 
-from pycparser import c_parser, c_generator, c_ast
+from pycparser import c_parser, c_generator, c_ast, parse_file
+
+CPPPATH = 'cpp'
```

Comment:
I prefer not to use `cpp` in these tests and have chunks of preprocessed code instead

Reply:
I've removed cpp!